premake5.lua

```diff
@@ -85,8 +85,14 @@ group "Game"
     kind "WindowedApp"
     configure_project_base()
     includedirs { "third_party/SDL/include/", "include" }
-    files { "src/**.cpp", "src/**.h" }
+    files { "src/**.cpp", "include/**.h", "include/**.hpp" }
```

Comment:
I would prefer if we just stuck to `.h` and not include any .hpp files. Let them be treated as "errors"